dashboard/Makefile

```diff
@@ -0,0 +1,24 @@
+GOCMD ?= go
+GOARGS= -trimpath
+REPLICATOR_OUTPUT_DIR ?= bin
+
+replicator: fmt vet
```

Comment:
1. Make targets have dashes by convention, so "configure-replicator", "configure-grafana".
2. I think it would make sense to move the default configuration into the dashboard directory structure, so config/dashboard -> dashboard/config/default. Under that, dashboards -> grafana to make the copying more logical.
3. We should have environment variables to specify config template locations. CONFIG_TEMPLATE_DIR ?= dashboard/config/default would be a possibility, or specific ones for grafana and the replicator.
4. I would reorder the targets to put grafana first and replicator second (coincidentally, that is alphabetical order).

Reply:
Since we have a file that comes from a deployment-specific configuration (config.yaml) and one that is generic (app.yaml), we could perhaps organize the configurations like this:

dashboard/config/postgres_replicator/app.yaml
dashboard/config/postgres_replicator/default/config.yaml

Similarly we can have

dashboard/config/grafana/app.yaml ...
dashboard/config/grafana/dashboards/default/...

The common parts can be copied from a fixed location onto the destination by make, with whatever substitutions are required. The "specific" are then copied from the location given by environment variables like CONFIG_TEMPLATE_DIR.

We should also consider whether the postgres replicator needs any substitution at all. The substitutions could all be made by the go code using the environment variables available when the code is deployed.

Reply:
1. Changed underscores to dashes in the Makefile
2. Configuration has been moved into `dashboard/config` as discussed
3. The environment variable `CONFIG_TEMPLATE_DIR` points to the configuration directory and is used in the Makefile
4. Targets reordered